src/core/wb.js

```diff
@@ -561,17 +569,18 @@ Modernizr.load( [
 
 					// Load the MathML dependency. Since the polyfill is only loaded
 					// when !Modernizr.mathml, we can skip the test here.
-					Modernizr.load( {
-						load: "http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=Accessible",
-						testReady: function() {
-							return ( window.MathJax && window.MathJax.isReady );
-						},
+					Modernizr.load( [ {
+						load: "timeout=500!https://cdn.jsdelivr.net/npm/mathjax@BOWER_VERSION_MATHJAX/MathJax.js?config=Accessible",
 						complete: function() {
+							Modernizr.load( [ {
+								test: window.MathJax === undefined,
+								yep: "mthjx!MathJax.js?config=Accessible"
+							} ] );
 
 							// Identify that initialization has completed
 							wb.ready( $document, componentName );
```

Comment:
This is still a potential race condition since you are not calling this line from the backup

Reply:
@nathanv Any news on Laurent's review?

Reply:
@nathanv To avoid the race condition you can run a test with ```testReady``` like:
```js
Modernizr.load( {
	load: "http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=Accessible",
	testReady: function() {
		return ( window.MathJax && window.MathJax.isReady );
	},
	complete: function() {

		// Identify that initialization has completed
		wb.ready( $document, componentName );
	}
} );
```
 https://github.com/wet-boew/wet-boew/blob/master/src/core/wb.js#L566